plugins/labhub.py

```diff
@@ -1,7 +1,104 @@
-from errbot import BotPlugin
+import json
+import os
+import re
 
+import github3
 
-class Labhub(BotPlugin):
-    """
-    GitHub and GitLab utilities
-    """
+from errbot import BotPlugin, re_botcmd
+
+
+class LabHub(BotPlugin):
+    """GitHub and GitLab utilities"""  # Ignore QuotesBear
+
+    def activate(self):
+        super().activate()
+
+        with open('plugins/LabHub.json') as f:
```

Comment:
I don't think there is a better solution then `os.path.join(os.environ.get('COBOT_ROOT', os.getcwd()), 'plugins/LabHub.json')`